pkg/bootstrapper/bootstrapper.go

```diff
@@ -522,15 +543,13 @@ func (wmcb *winNodeBootstrapper) ensureKubeletService() error {
 		Password:         "",
 		Description:      "OpenShift Kubelet",
 	}
-	// Get kubelet args
-	kubeletArgs := wmcb.getInitialKubeletArgs()
 
 	if wmcb.kubeletSVC == nil {
-		if err := wmcb.createKubeletService(c, kubeletArgs); err != nil {
+		if err := wmcb.createKubeletService(c, wmcb.kubeletArgs); err != nil {
```

Comment:
If you are making the actual `kubeletArgs` part of the `winNodeBootstrapper`, it should not be passed as an argument to a struct method